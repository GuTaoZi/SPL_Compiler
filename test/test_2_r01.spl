int test_2_r01(int a, int b)
{
  int c='c';
  int x=-1.1+ +2.2;
  int y=-123;
  x = x-5;
  y=y+-5;
}
