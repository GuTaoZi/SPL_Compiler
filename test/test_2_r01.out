Error type A at Line 4: unknown lexeme 1...1
Error type A at Line 5: unknown lexeme $
Error type B at Line 5: Missing semicolon ';'
